$OpenBSD: patch-SConstruct,v 1.1 2009/04/04 14:53:38 ajacoutot Exp $

Needed with newer scons to avoid the following error:
scons: *** [installtarget] AttributeError : 'NoneType' object has no attribute 'genstring'

Should not be needed when widelands gets updated.

--- SConstruct.orig	Sat Apr  4 16:35:45 2009
+++ SConstruct	Sat Apr  4 16:35:55 2009
@@ -305,7 +305,6 @@ instadd(env, 'widelands', filetype='binary')
 install=env.Install('installtarget', 'build-widelands.sh') # the second argument is a (neccessary) dummy
 Alias('install', install)
 AlwaysBuild(install)
-env.AddPreAction(install, Action(buildlocale))
 
 uninstall=env.Uninstall('uninstalltarget', 'build-widelands.sh') # the second argument is a (neccessary) dummy
 Alias('uninstall', uninstall)
